backend/app/tasks.py

```
@@ -4701,3 +4712,419 @@ def _record_webhook_notification(item, portal_instance):
+                    backup.status = "applied"
+                    if is_stale:
+                        backup.error = ("Item was modified in the portal after the dry run; "
+                                        "re-analyzed and updated from its current content.")
+                    # Snapshot the post-update modified time so a later revert
+                    # can detect (and flag) edits made after the replacement.
+                    # 0 = snapshot failed: the revert flow must treat the item
+                    # as possibly edited rather than silently skip the check
+                    try:
+                        refreshed = target.content.get(item.id)
+                        backup.applied_modified_at = getattr(refreshed, "modified", None) or 0
+                    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)